static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx

```diff
@@ -42,7 +43,22 @@ export default function FeatureFlagsPanel() {
           .map(flag => {
             return (
               <Cell key={flag} style={{alignItems: 'flex-start'}}>
-                {flag}
+                {featureFlagTemplateUrl ? (
+                  <ExternalLink
+                    style={{textDecoration: 'inherit', color: 'inherit'}}
+                    href={featureFlagTemplateUrl(flag)}
```

Comment:
Might need to be {featureFlagTemplateUrl(flag) ? (`

so we catch a case where the empty string is returned. or maybe the return type should be `string | undefined` for the function? idk.

Reply:
not super important tho

Reply:
just modified !